index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <script src="https://kit.fontawesome.com/4a30d59e86.js" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="./styles/style.css" />
+        <link rel="stylesheet" href="./styles/contact.css" />
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet"
+            integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT"
+            crossorigin="anonymous" />
+        <title>International Conference on Artificial Intelligence, Machine Learning and Big Data Engineering</title>
+    </head>
+</head>
+
+<body>
+    <header>
+        <div class="top-header">
+            <div class="top-links-container d-flex align-items-center">
+                <i class="fa-brands fa-facebook-f"></i>
+                <i class="fa-brands fa-twitter"></i>
+                <p class="m-0">English</p>
+                <p class="m-0">My Page</p>
+                <p class="m-0">Logout</p>
+            </div>
+        </div>
+        <div class="d-flex justify-content-between align-items-center navbar-style px-4">
+            <a class="logo" href="./index.html">
+                <img src="https://isete.org/images/logo.png" alt="ISETE Logo" />
```

Comment:
- [ ] Good job so far I can see that on the right side of the nave we do have the name of the company, kindly complete that part for your project to match the template design :wink: ,  Also, for the bottom navbar, please make sure that the margin on the left-hand side and right-hand side are equal. 
Your design :point_down: 
![Screenshot from 2023-04-20 10-04-35](https://user-images.githubusercontent.com/88894525/233301566-519f84a5-7d08-44a0-9288-e8e81a2c5836.png)

Template design :point_down: 
![dyjxrtj](https://user-images.githubusercontent.com/73607512/232623047-43534337-6836-460d-9010-bd62bf960822.PNG)

_note: I can see that you are missing some pages on your nave bar such as Pograms,.._